packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -20,28 +17,13 @@ import { ChangePasswordPage } from '../../pages/settings/changePassword';
 // Disable this check for these tests. We are holding assertion in shared functions due
 // to how test permutations work, and these setup falsely trips this rule.
 /* eslint-disable playwright/expect-expect */
+const AGE_21 = '21';
```

Comment:
Mabye the eslint-disable line above this can be removed?

Reply:
I think it's best left as a separate PR. The structure of this test module isn't well understood by the linter.
<img width="564" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/c3aaa44c-812e-487d-b5a0-53a9dfe49337">